biostudiesclient/auth.py

```diff
@@ -0,0 +1,76 @@
+"""
+biostudiesclient.auth
+~~~~~~~~~~~~
+
+This module dealing with authentication
+
+:copyright: (c) 2021 by Karoly Erdos.
+:license: Apache2, see LICENSE for more details.
+"""
+
+
+from dataclasses import dataclass
+from http import HTTPStatus
+import requests
+
+from biostudiesclient.config import BIOSTUDIES_USERNAME, BIOSTUDIES_PASSWORD, BIOSTUDIES_API_URL
+from biostudiesclient.response_utils import STATUS_CODE_OK
+from biostudiesclient.response_utils import ResponseUtils
```

Comment:
this can be simplified to:
`from biostudiesclient.response_utils import ResponseUtils, STATUS_CODE_OK`

Reply:
Done and pushed